charactersheet/charactersheet/models/common/armor.js

```diff
@@ -25,7 +25,12 @@ function Armor() {
     self.armorStealthOptions = ko.observableArray(Fixtures.armor.armorStealthOptions);
 
     self.acLabel = ko.pureComputed(function() {
-        return 'AC ' + self.armorClass();
+        if (self.armorClass().replace(/^\s+|\s+$/g, '')) {
```

Comment:
Oh this is that issue. If we care, we can polyfill it in older browsers like [they do here](http://stackoverflow.com/questions/498970/trim-string-in-javascript#498995) otherwise leave it I guess.

Thoughts?